cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -164,12 +164,38 @@ <h2 class="h3 u-mb15">
             {{ svg_icon("cart-round") }}
             Making purchases
         </h2>
-        {# How top-line APR displays if the card has:
-            - Min and max APRs: "19.49% - 31.99% purchase APR"
-            - No min and max but a median: "29.74% median purchase APR"
-            - Single APR for everyone: "24.90% purchasae APR"
-        #}
-        {% if card.purchase_apr_offered %}
+        {# TODO: Unify ratings code with results list view #}
+        {% set purchase_apr_rating_labels = [
+            'less', 'average', 'more'
+        ] %}
+        {% set purchase_apr_ratings = [
+            card.purchase_apr_poor_rating,
+            card.purchase_apr_good_rating,
+            card.purchase_apr_great_rating
+        ] %}
```

Comment:
Would it be easier if we serialized the APRs like this, e.g. the JSON would have something like

```json
{
  "purchase_aprs": {
    "good": [0.99, 0.99],
    "great": [0.79, 0.79],
    "poor": [0.6, 0.6]
  }
}
```

Reply:
@chosak, something like that would be helpful. What do you think about holding off on that change for now and taking another look at how we serialize things more broadly in a future PR? I could also see serializing the cash advance APR the same way we do the other two APRs so we can use a consistent structure for all the APR tables on this page, so maybe it would make sense to do all the re-serializing at once.